### ISSUE TITLE
Fail to restore third window which name starts with digit

### ISSUE THREAD

**Issue by soliosg** [tmux-plugins/tmux-resurrect#58]
Commands used to save tmux-resurrect environment:
```
tmux new-session -s work -n home
tmux new-window -n 3632
tmux new-window -n 2981
tmux new-window -n 3209
tmux select-window -t home
C-b C-s
```

Status bar after saving:
`[work] 1:home* 2:3632 3:2981 4:3209-`

Steps to close tmux, start it and restore the environment:
```
    tmux detach
    tmux kill-server
    tmux new-session
    C-b C-r
```

Status bar after restore has missing the third window, `3:2981`:
`[work] 1:home* 2:3632 4:3209-`

Here is the contents of tmux_resurrect_*timestamp*:
```
pane	work	1	:home	1	:*	0	:/home/soliosg	1	bash	:
pane	work	2	:3632	0	: 	0	:/home/soliosg	1	bash	:
pane	work	3	:2981	0	: 	0	:/home/soliosg	1	bash	:
pane	work	4	:3209	0	:-	0	:/home/soliosg	1	bash	:
window	work	1	1	:*	b73d,191x52,0,0,0
window	work	2	0	: 	b73e,191x52,0,0,1
window	work	3	0	: 	b73f,191x52,0,0,2
window	work	4	0	:-	b740,191x52,0,0,3
state	work	
```

**Comment by soliosg** [comment]
A patch for this issue has been applied, http://sourceforge.net/p/tmux/tickets/169/. Hopefully, this will be included in tmux's 2.0 release.

**Comment by bruno-** [comment]
I just checked and this seems to be resolved in tmux 2.0
